q2_diversity/tests/test_procrustes.py

```diff
@@ -0,0 +1,114 @@
+                'PCoA',
+                'Principal Coordinate Analysis',
+                eigvals,
+                samples_df,
+                proportion_explained=proportion_explained)
+
+        samples = np.array([[0.7, 3.7, 4.7, 4.7, 0.7, 0.7],
+                            [1.7, 2.7, 1.7, 4.7, 3.7, 3.7],
+                            [2.7, 3.7, 1.7, 0.7, 0.7, 1.7],
+                            [30, 3.7, 2.7, 4.7, 3.7, 0.7]])
+        samples_df = pd.DataFrame(samples,
+                                  index=['A', 'B', 'C', 'D'],
+                                  columns=axes)
+        self.other = skbio.OrdinationResults(
+                'PCoA',
+                'Principal Coordinate Analysis',
+                eigvals.copy(),
+                samples_df.copy(),
+                proportion_explained=proportion_explained.copy())
+
+        S = [[-0.1358036, 0.0452679, 0.3621430, 0.1810715, -0.2716072],
+             [0.0452679, -0.1358036, -0.1810715, 0.1810715, 0.2716072],
+             [0.2263394, 0.0452679, -0.1810715, -0.5432145, -0.2716072],
+             [-0.1358036, 0.0452679, 0.0000000, 0.1810715, 0.2716072]]
+        samples_df = pd.DataFrame(np.array(S),
+                                  index=['A', 'B', 'C', 'D'],
+                                  columns=axes[:5])
+        self.expected_ref = skbio.OrdinationResults(
+                'PCoA',
+                'Principal Coordinate Analysis',
+                eigvals[:5].copy(),
+                samples_df.copy(),
+                proportion_explained=proportion_explained[:5].copy())
+        S = [[0.0482731, -0.0324317, 0.0494312, -0.0316828, -0.1584374],
+             [0.0803620, -0.0718115, -0.0112234, -0.0171011, -0.1101209],
+             [0.0527554, -0.0042753, -0.0126739, -0.0969602, -0.0964822],
+             [-0.1813905, 0.1085184, -0.0255339, 0.1457440, 0.3650405]]
+        samples_df = pd.DataFrame(np.array(S),
+                                  index=['A', 'B', 'C', 'D'],
+                                  columns=axes[:5])
+        self.expected_other = skbio.OrdinationResults(
+                'PCoA',
+                'Principal Coordinate Analysis',
+                eigvals[:5].copy(),
+                samples_df.copy(),
+                proportion_explained=proportion_explained[:5].copy())
+
+    def test_procrustes(self):
+        ref, other = procrustes_analysis(self.reference, self.other)
+
+        skbio.util.assert_ordination_results_equal(ref, self.expected_ref)
+        skbio.util.assert_ordination_results_equal(other, self.expected_other)
+
+    def test_procrustes_bad_dimensions(self):
+
+        self.other.samples = self.other.samples.iloc[:, :4]
+        self.other.eigvals = self.other.eigvals[:4]
+        self.other.proportion_explained = self.other.proportion_explained[:4]
+
+        with self.assertRaisesRegex(ValueError, 'The matrices cannot be '):
+            _ = procrustes_analysis(self.reference, self.other)
```

Comment:
`flake8` is mad about the unused variable :disappointed:. It's a shame, I really like this style.